### DISCUSSION TITLE
Enabling/Disabling multiple devices not working properly

### DISCUSSION THREAD

**Discussion by bdf0506** [frenck/spook#526] [Q&A] [answered]
According to https://spook.frenck.dev/devices#disable-a-device, you should be able to use a service call to enable/disable multiple devices.

But I can't seem to get that to work:

````
service: homeassistant.disable_device
data:
  device_id: 
    - "aad6033acabdea5f75f19af03311ed6d"
    - "d878bbe018cdffff352fa62b256be8e9"
````

results in:

`
2024-01-03 14:05:53.195 ERROR (MainThread) [homeassistant.helpers.script.websocket_api_script] websocket_api script: Error executing script. Invalid data for call_service at pos 1: value should be a string for dictionary value @ data['device_id']
`

Calling a single device works fine though.

**Comment by bdf0506** [comment, marked as answer]
Nevermind. I was on 1.3.2, looks like this now is supported in 1.4.0

**Reply by frenck** [reply]
Correct! This was added later 👍
